RecoPixelVertexing/PixelVertexFinding/src/PixelVertexHeterogeneousConverter.cc

```diff
@@ -0,0 +1,55 @@
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "HeterogeneousCore/Product/interface/HeterogeneousProduct.h"
+
+#include "RecoPixelVertexing/PixelVertexFinding/interface/pixelVertexHeterogeneousProduct.h"
+
+
+class PixelVertexHeterogeneousConverter: public edm::global::EDProducer<> {
```

Comment:
The "heterogeneous producer" produces already `reco::VertexCollection`, so is this converter needed?

(I'm hoping we get to the next framework before we have a GPU client for pixel vertices)

Reply:
see below...